doc/python/probe_sample.rst

```diff
@@ -5,23 +5,66 @@ Cable cell probing and sampling
 
 .. module:: arbor
 
-Cable cell probe addresses are defined analogously to their counterparts in
-the C++ API (see :ref:`cablecell-probes` for details). Sample data recorded
-by the Arbor simulation object is returned in the form of a NumPy array,
-with the first column holding sample times, and subsequent columns holding
-the corresponding scalar- or vector-valued sample.
-
-Location expressions will be realised as zero or more specific sites on
-a cell; probe addresses defined over location expressions will describe zero,
+.. figure:: probe_sample-diag.svg
+    :width: 800
+    :align: center
+
+    A schematic view of how :term:`handles <handle>` let you access sampled data measured at a :term:`probeset`.
+    When setting a probe on a locset a probeset is created (a locset describes one or more locations).
```

Comment:
> When setting a probe on a locset a probeset is created (a locset describes one or more locations).

`probe` is already used in the schema (and everywhere else) to mean "part of a probeset", and here it is used as "thing that combined with a locset makes a probeset". We are trying to avoid this. Is it an option call the functions `probing devices`/`probing functions`/`probe objects` rather than `probes`? Alternatively we can call the thing inside of a probeset a `probeloc`?

Reply:
Anyways, we could also leave it like this and rewrite / note it:

```
When setting a probe object on a locset, a probeset is created (a locset describes one or more locations).
Note that both the probe objects returned by probe functions, and the elements of a probeset may be referred to as probes.
```

Reply:
I also noticed below that we use `probeset functions`, maybe the solution is just to `probeset function` instead of `probe` here?

Reply:
`probe` and `sample_recorder` (which is ultimately what arbor calls a thing that records samples) are not sufficiently distinguished I guess. 

When we place a probe(set) on a loc(set), we create a sample_recorder (no, not gonna define `sample_recorderset`!) Is this the same as a `sampler`? Probably, @thorstenhater can you confirm?

Reply:
I can confirm.